zapcli/zap_helper.py

```diff
@@ -394,5 +394,5 @@ def html_report(self, file_path):
     @staticmethod
     def _write_report(report, file_path):
         """Write report to the given file path."""
-        with open(file_path, 'w') as f:
+        with open(file_path, mode='wb') as f:
             f.write(report)
```

Comment:
Hmm, this worked for me using your Gist in Python 2.7:
```python
import urllib2
response = urllib2.urlopen('https://gist.githubusercontent.com/omerlh/a974942028e744760374b62bca57fc0e/raw/0c4bb18caf00eaf336878f8cb56d630ca3a7616a/report.html')
report = response.read()
with open('test2.html', 'wb') as f:
    f.write(report.encode('utf-8'))
```

Let me know if there's another test page where this breaks in Python 2.7.